ui/src/app/pages/admin/admin-institution.tsx

```diff
@@ -0,0 +1,153 @@
+      case OrganizationType.HEALTHCENTERNONPROFIT: return 'Health Center Non-Profit';
+      case OrganizationType.EDUCATIONALINSTITUTION: return 'Educational Institution';
+      case OrganizationType.INDUSTRY: return 'Industry';
+      default: return 'Other';
+    }
+  }
+
+  renderDuaType(row, col) {
+    return row['duaTypeEnum'] === DuaType.RESTRICTED ? 'Individual' : 'Master';
+  }
+
+  // If email domain list has more than 4 entries show top 4 and replace others with ...
+  renderEmailDomain(row, col) {
+    const emailDomain = fp.take(4, row['emailDomains']).join('\n') ;
+    if (row['emailDomains'] && row['emailDomains'].length > 4) {
+      return emailDomain + '...';
+    }
+    return emailDomain;
+  }
+
+  // If email address list has more than 4 entries show top 4 and replace others with ...
+  renderEmailAddress(row, col) {
+    const emailAddresses = fp.take(4, row['emailAddresses']).join('\n') ;
+    if (row['emailAddresses'] && row['emailAddresses'].length > 4) {
+      return emailAddresses + '...';
+    }
+    return emailAddresses;
+  }
+
+  render() {
+    const {institutions, institutionLoadError, loadingInstitutions} = this.state;
+    return <div>
+      <FadeBox style={{marginTop: '1rem', marginLeft: '1rem'}}>
+        <SemiBoldHeader style={styles.pageHeader}>
+          <label>Institution admin table</label>
+              <Button type='secondaryLight'
+                      style={{padding: '0rem', marginTop: '0.3rem', verticalAlign: 'sub'}}
+                      data-test-id='add-institution'>
+                <ClrIcon shape='plus-circle' class='is-solid' size={20}/>
+              </Button>
+        </SemiBoldHeader>
+        {institutionLoadError && <div style={{color: colors.danger}}>
+          Error while loading Institution please try again later</div>}
+        <DataTable data-test-id='institution-datatable' value={institutions} paginator={true}
+                   rows={5} scrollable={true} frozenWidth='7rem' loading={loadingInstitutions}>
+          <Column field='displayName' header='Institution Name'
+                  bodyStyle={styles.text} headerStyle={styles.header} frozen={true}/>
+          <Column field='organizationTypeEnum' header='Institution Type'
+                  body={this.renderOrganizationType} bodyStyle={styles.text}
+                  headerStyle={styles.header}/>
+          <Column field='duaTypeEnum' header='Agreement Type' body={this.renderDuaType}
+                  bodyStyle={styles.text} headerStyle={styles.header}/>
+          <Column field='emailDomains' header='Accepted Domain List' body={this.renderEmailDomain}
+                  bodyStyle={styles.text} headerStyle={styles.header}/>
+          <Column field='emailAddresses' header='Accepted Email List' body={this.renderEmailAddress}
+                  bodyStyle={styles.text} headerStyle={styles.header}/>
+          <Column field='userInstructions' header='User Instruction' bodyStyle={styles.text}
```

Comment:
Something indicating that this refers to an email.  Maybe "User Email Instructions"